web/MetaInfo.py

```diff
@@ -8,7 +9,9 @@ def __init__(self):
         Initializes MetaInfo object with meta language information
         :rtype: None
         """
-        with open("web/thesauruses/meta_info.json", 'r') as meta_file:
+        meta_info_file_path = os.path.join(
```

Comment:
omg more variables! this thing is already complex!

(though I approve of this change, lol)